plugins/chat/assets/javascripts/discourse/lib/hashtag-types/channel.js

```diff
@@ -20,7 +20,7 @@ export default class ChannelHashtagType extends HashtagTypeBase {
 
   generateColorCssClasses(channel) {
     return [
-      `.hashtag-cooked .d-icon.hashtag-color--${this.type}-${channel.id} { color: var(--category-${channel.chatable.id}-color); }`,
+      `.d-icon.hashtag-color--${this.type}-${channel.id} { color: var(--category-${channel.chatable.id}-color); }`,
```

Comment:
I removed hashtag-cooked as I don't think we need this specificity and would have forced me to write more css for `.hashtag-autocomplete__link`

Reply:
Yes good point, ty 👍